core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -7578,6 +7578,22 @@ private void checkLiteral2(String expression, String expected) {
     sql(query).withPresto().ok(expected);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6892">[CALCITE-6892]
+   *  CHAR_LENGTH Function is not recognized in DerbySQL.
```

Comment:
Here it remains inconsistent. Please double-check.

Reply:
Thanks, had changed, ptal~ @NobiGo